docs/index.md

```diff
@@ -137,26 +137,26 @@ custom: {
     </div>
     <div class="Fl(start) W(60%) Cl(b) Fl(n)--xs W(a)--xs">
 {% highlight html %}
-<div class="Bd Bgc(#0280ae):h C(#0280ae) C(#fff):h P(20px)">
+<div class="Bgc(#0280ae):h C(#0280ae) C(#fff):h">
```

Comment:
assume these are ok...

Reply:
I left the examples with hardcoded hex to make it clearer to users. I didn't want to complicate it with CSS vars.